solutions/USACO feb20-swap.mdx

```diff
@@ -242,3 +246,52 @@ int main() {
 	for (int t: cur) cout << t+1 << "\n";
 }
 ```
+
+</CPPSection>
+
+<PySection>
+
+## Python Implementation
+
+**Time Complexity:** $\mathcal{O}(NK)$
+
+```py
+# Take in the input using standard Python file i/o system
+file_in = open('swap.in')
+data = file_in.read().strip().split('\n')
+n, k = map(int, data[0].split(' '))
+swaps = [list(map(int, data[i+1].split(' '))) for i in range(2)]
```

Comment:
add whitespace between operators please

Reply:
where? this seems fine...?

Reply:
```suggestion
swaps = [list(map(int, data[i + 1].split(' '))) for i in range(2)]
```

Reply:
and in most other places, maybe op should just put it through a pycharm formatter